### ISSUE TITLE
Remove the default class .linkified

### ISSUE THREAD

**Issue by Hongbo-Miao** [Hypercontext/linkifyjs#216]
Thanks for the great package!

It would be great to have option to remove the default class `.linkified`.

Right now neither of these two works.
```js
linkifyHtml(html, {
  className: ''
});
```

```js
linkifyHtml(html, {
  className: null
});
```

**Comment by groe** [comment]
:+1:

Workaround:
```js
linkify.options.defaults.className = "";
```

**Comment by Hongbo-Miao** [comment]
Thank you @groe, I will try tonight!

**Comment by Hongbo-Miao** [comment]
Hi @groe how did you import?

**1.**
```
import linkifyHtml from 'linkifyjs/html';
linkifyHtml.options.defaults.className = '';
```
gives me error
> TypeError: Cannot read property 'defaults' of undefined

**2.**
```
import linkify from 'linkifyjs';
linkify.options.defaults.className = '';
```
gives me error
> TypeError: Cannot read property 'options' of undefined

**Comment by groe** [comment]
@Hongbo-Miao I don't import it, I just include it in the page. Basically it's a `<script src="linkify.min.js"></script>`

**Comment by nfrasser** [comment]
This will be fixed in Linkify 3.0

**Comment by nfrasser** [comment]
Fixed in v3
